### PR TITLE
[Merged by Bors] - feat: port Data.Set.Intervals.WithBotTop

### PR DESCRIPTION
**Pull request by mcdoll**

### CODE REVIEW

**Comment by mcdoll** [comment]
@ChrisHughes24 @semorrison I am confused, Scott said that I don't have to add the hash, so what is the label about Chris?

**Comment by ChrisHughes24** [comment]
I would trust Scott, I didn't get the update that hashes are no longer necessary.

File: Mathlib/Data/Set/Intervals/WithBotTop.lean [hunk 1]
```diff
@@ -0,0 +1,237 @@
+/-
+Copyright (c) 2022 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+! This file was ported from Lean 3 source module data.set.intervals.with_bot_top
```
Comment (jcommelin) [inline Mathlib/Data/Set/Intervals/WithBotTop.lean:5]:
```suggestion

! This file was ported from Lean 3 source module data.set.intervals.with_bot_top
```

**Comment by dupuisf** [comment]
LGTM

bors r+

**Pull request closed without merging**